portfolio/src/main/webapp/style.css

```diff
@@ -1,16 +1,71 @@
+/* GENERAL PROPERTIES */
+
+html, body {
+    height: 100%;
+    margin: 0;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    height: 100%;
+}
+
+/* HEADER PROPERTIES */
+
+#header {
+    background-color: azure;
+    height: 250px;
+}
+
+#titleWrapper {
+    height: 75%;
+}
+
+#tabSelector {
+    height: 25%;
+    margin-right: 18px;
+}
+
+.tab {
+    background-color: aliceblue;
+    width: 15%;
+    height: 100%;
+    float: right;
```

Comment:
This will show the elements in the opposite order of what's in the HTML. Another approach is to wrap your tabs in a div and float that div to the right instead of floating each tab.

Reply:
Good point, that's a lot more straightforward.